### ISSUE TITLE
关于博主新旧SSD算法对比

### ISSUE THREAD

**Issue by lssttxs** [bubbliiiing/ssd-pytorch#58]
博主，为什么我用你最初写的SSD版本训练自己的数据集（只有500张左右）得到的map值比最新的这一版的效果更好呢？

**Comment by bubbliiiing** [comment]
sgd训练要求更高，数据集少可以用adam

**Comment by lssttxs** [comment]
嗯，我按照注释里的，优化器选成了adam学习率下降方式选的step，map大概61%，老版的map大概有68%，为什么降低这么多呢

**Comment by bubbliiiing** [comment]
新版的用sgd效果会好很多。
你的其它学习策略呢？

**Comment by bubbliiiing** [comment]
sgd在大数据上表现会好一些，更鲁棒一些，但小数据集上表现一般…

**Comment by bubbliiiing** [comment]
噢对，还有你的数据集情况也有影响的，数据集检测什么？

**Comment by lssttxs** [comment]
如果是两千张的数据集，适用sgd么？2000张是不是也很小。。其他学习策略是指？

**Comment by bubbliiiing** [comment]
……检测啥，之类的，都有影响，我数据增强都改了。

**Comment by bubbliiiing** [comment]
学习率？weight decay？

**Comment by lssttxs** [comment]
我检测火焰。只检测火焰，我的标签只有火焰。

**Comment by lssttxs** [comment]
学习率就是按博主注释里的对应着改了，然后adam下weight decay设置成了0

**Comment by lssttxs** [comment]
那是不是可能我把数据增强部分改成适合火焰的数据增强方法，新版本效果会变好

**Comment by bubbliiiing** [comment]
适合火焰的是什么……
你的图片高宽比例是否正常？

**Comment by bubbliiiing** [comment]
你试过sgd么
